src/components/gauge/gauge.e2e.ts

```diff
@@ -0,0 +1,156 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+
+describe("gx-gauge", () => {
+  let element: E2EElement;
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+  });
+
+  it("should be drawn w/o attrs", async () => {
+    await page.setContent("<gx-gauge></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    expect(element).toBeTruthy();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  });
+  /////////////////// Type test ///////////////////////
+
+  it("should render with the line type", async () => {
+    await page.setContent("<gx-gauge gauge-type='Circle'></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await element.setAttribute("gauge-type", "Line");
+    await page.waitForChanges();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  });
+
+  it("should render with the circle type", async () => {
+    await page.setContent("<gx-gauge gauge-type='Circle'></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find("svg")).toBeTruthy();
+  });
+
+  /////////////////// Values test ///////////////////////
+  it("should set min value", async () => {
+    await page.setContent(
+      "<gx-gauge gauge-type='Circle' min-value='0'></gx-gauge>"
+    );
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find("svg")).toBeTruthy();
+    expect(await element.getProperty("minValue")).toEqual(0);
```

Comment:
I'd like to see some expectations on the effect of setting the `minValue` other than if the value is set in the property.